tests/unit/bokeh/core/test_properties.py

```diff
@@ -326,10 +311,10 @@ class Readonly(HasProps):
         assert 'y' in o.properties()
         assert 'z' in o.properties()
 
-        # but they aren't in the dict of props with values
```

Comment:
@mattpap please note none of these tests were running at all because the class was name `BasicTest` instead of `TestBasic` Most were obvious to fix up but note this comment was not true, readonly properties *do* show up in `properties_with_values` presently. I think that is fine, but I wanted to bring your attention to it.